api/agent.go

```diff
@@ -162,10 +168,19 @@ func (a *Agent) MembersOpts(opts *QueryOptions) (*ServerMembers, error) {
 }
 
 // ForceLeave is used to eject an existing node from the cluster.
-func (a *Agent) ForceLeave(node string, prune bool) error {
+func (a *Agent) ForceLeave(node string) error {
+	_, err := a.client.put("/v1/agent/force-leave?node="+node, nil, nil, nil)
```

Comment:
We've dropped the URL encoding we were doing previously on the values. Let's make sure we're still doing that here.